package/MDAnalysis/visualization/streamlines.py

```diff
@@ -85,7 +86,8 @@ def produce_grid(tuple_of_limits, grid_spacing):
     return grid
 
 
-def split_grid(grid, num_cores):
+# typing: numpy
+def split_grid(grid: np.ndarray, num_cores: int) -> List[List[List[int]], List[List[int]], int, int]:
```

Comment:
A list does not let you annotate each value of it, only a tuple does. There is no reason for this function to return a list; returning a tuple should be OK.